helpers/markdownDocumentsReader.ts

```diff
@@ -18,16 +18,22 @@ export function getPostDocumentBySlug(slug: string) {
   return JSONSerialize({ slug: realSlug, ...data, content }) as PostDocument;
 }
 
+const today = new Date();
+
+const isPostInTheFuture = (posts: PostDocument[]) =>
+  posts.filter((item) => today.toISOString() > item.date);
```

Comment:
```suggestion
const isPostInTheFuture = (post: PostDocument) =>
  if ((post.date) => today.toISOString() > item.date) {
    return true;
  } else {
    return false;
  }
```

@ntorbinskiy is it clear now?